src/orderbook.rs

```diff
@@ -83,7 +83,15 @@ pub enum OrderState {
     REJECT,
 }
 
-// Create Order
+/// Used for order upate events after an order has been successfully matches
+/// and all parties need to be notified
+#[derive(Debug, Clone, Encode, Decode, PartialEq)]
+pub struct OrderUpdate {
+    pub order: Order,
```

Comment:
We need status field for the order

Reply:
Should this be incuded directly in the `Order` strcut or in the `OrderUpdate` struct?

Reply:
Ah I got it. I'll add an OrderStatus enum